src/django/api/tests/test_wage_indicator_provider.py

```diff
@@ -0,0 +1,238 @@
+from django.test import TestCase
+from django.contrib.gis.geos import Point
+from django.core.cache import cache
+
+from api.models import (
+    Contributor,
+    Facility,
+    FacilityList,
+    FacilityListItem,
+    FacilityMatch,
+    Source,
+    User,
+)
+from api.models.partner_field import PartnerField
+from api.models.wage_indicator_country_data import WageIndicatorCountryData
+from api.models.wage_indicator_link_text_config import (
+    WageIndicatorLinkTextConfig
+)
+from api.partner_fields.wage_indicator_provider import WageIndicatorProvider
+
+
+class WageIndicatorProviderTest(TestCase):
+    def setUp(self):
+        '''Set up test data for wage indicator provider tests.'''
+        self.email = 'test@example.com'
+        self.user = User.objects.create(email=self.email)
+        self.contributor = Contributor.objects.create(
+            admin=self.user,
+            name='Test Contributor',
+            contrib_type=Contributor.CONTRIB_TYPE_CHOICES[0][0],
+        )
+
+        # Get or create wage_indicator partner field.
+        # Use get_all_including_inactive to access existing system field.
+        try:
+            self.partner_field = PartnerField.objects \
+                .get_all_including_inactive() \
+                .get(name='wage_indicator')
+        except PartnerField.DoesNotExist:
```

Comment:
Probably wort it to add logging for this specific `except` statement to make sure we understand when this behaviour happens.